packages/twenty-e2e-testing/playwright.config.ts

```diff
@@ -56,6 +56,10 @@ export default defineConfig({
       },
       dependencies: ['Login setup'],
     },
+    {
+      name: 'Authentication',
+      testMatch: /authentication\/.*\.spec\.ts/,
+    },
```

Comment:
I'm unsure about the use of Playwright projects here. To me, this feature is useful to run projects on different environments. Wouldn't it be more straightforward to run authentication tests by specifying the path to the folder where you extracted them? What do you thin?

Reply:
I use projects to split tests by things being tested, I can change it to tags inside tests so there won't be need to use projects for them but I'll think about it